### ISSUE TITLE
a11y-view broken for builds

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/chipper#649]
Source of the iframe is `'{{PHET_SIM_URL}}' + simulationQueryString`. This doesn't work with the a11y-view generated and placed in build directory.

**Comment by jessegreenberg** [comment]
Fixed with the above commits.
